tsconfig.json

```diff
@@ -12,9 +12,11 @@
     "noUnusedParameters": true,
     "rootDir": "./src",
     "pretty": true,
+    "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
     "target": "es5"
   },
+  "include": ["src/types/**/*.d.ts"],
```

Comment:
we have declared types within `src/types/**/*.d.ts` that were not recognized since they were not included to `tsconfig.json`